docs/src/tutorials/dependency-updates.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Automatically update your application dependencies
+sidebarTitle: Update your application dependencies
```

Comment:
This is more of a suggestion, since the tutorial really is about setting up the cron + notifications to do this automatically, rather than just updating things

Reply:
@GuGuss @marcoporri Let us know what you think?

Reply:
@chadwcarlson @AnouckColson so we keep "Automatically update your application dependencies" as a page title and use "Update your application dependencies" as a sidebar title?

Reply:
If yes, then I'm not so sure. I understand where you're coming from, however using "schedule" instead of "update" seems less engaging to me

Reply:
OK, makes sense! Worst case scenario, search bar will still return the right pages when looking for "update dependency". @AnouckColson we can go for @chadwcarlson 's suggestion :)